.github/workflows/test.yml

```diff
@@ -432,7 +432,7 @@ jobs:
       OVN_EMPTY_LB_EVENTS: "${{ matrix.target == 'control-plane' || matrix.target == 'control-plane-helm' }}"
       OVN_HA: "${{ matrix.ha == 'HA' }}"
       OVN_DISABLE_SNAT_MULTIPLE_GWS: "${{ matrix.disable-snat-multiple-gws == 'noSnatGW' }}"
-      KIND_INSTALL_METALLB: "${{ matrix.target == 'control-plane' }}"
+      KIND_INSTALL_METALLB: "${{ matrix.target == 'control-plane' || matrix.target == 'control-plane-helm' }}"
```

Comment:
so IIUC you have with this bridged the gap of installing metalLB with helm lane, but are we running the load balancer services tests yet on helm lane? I think that is not yet enabled in this PR correct?

Reply:
Oh, it will run the test, @tssurya ! that is because the logic to skip running the test will see that this env variable is defined and not skip it any longer. You can confirm it by looking that the job. Good stuff! ;)

Ref:  https://github.com/ovn-org/ovn-kubernetes/blob/f5d4dfb6acef57d9a4cad95fedd672dfe4112f9a/test/scripts/e2e-cp.sh#L170

Reply:
I see and so now in all lanes we always run with METALLB = true right? this skip logic was added only for heml?
so do we still need that in e2e-cp.sh ? or shall we clean that up? @flavio-fernandes

Reply:
Yes, I think you are correct, but I do not see the harm in having the ability to skip tests we know will fail if `KIND_INSTALL_METALLB` is false. When running these tests locally, for instance, I do not install metal-lb and would be using that flag. So to me, it would make more sense if we were also removing the `KIND_INSTALL_METALLB` completely. Agree?